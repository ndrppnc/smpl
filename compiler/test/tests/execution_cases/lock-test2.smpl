int sum=0;
int add_to_sum(int val){
   lock sum = sum+val;
}

int main(){
   int i;
   pfor(4;i;0;10000){
      add_to_sum(i);
   }
   printf("The sum is %d.\n",sum);
}


