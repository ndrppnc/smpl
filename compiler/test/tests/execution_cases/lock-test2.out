The sum is 49995000.
